pkg/apis/operator/v1alpha1/tektonconfig_validation.go

```diff
@@ -43,6 +44,36 @@ func (tc *TektonConfig) Validate(ctx context.Context) (errs *apis.FieldError) {
 		}
 	}
 
+	// validate SCC config
+	if IsOpenShiftPlatform() && tc.Spec.Platforms.OpenShift.SCC != nil {
+		defaultSCC := PipelinesSCC
+		if tc.Spec.Platforms.OpenShift.SCC.Default != "" {
+			defaultSCC = tc.Spec.Platforms.OpenShift.SCC.Default
+		}
+
+		maxAllowedSCC := tc.Spec.Platforms.OpenShift.SCC.MaxAllowed
+		if maxAllowedSCC != "" {
+			// Check that maxAllowed SCC and default SCC are compatible wrt prioritiy
+			securityClient := utils.GetSecurityClient(ctx)
```

Comment:
We really need better package name than `utils` 😓; e.g. `securitycontext.GetClient(ctx)`, `securitycontext.GetPrioritizedList`, …

Reply:
one way is to put this under `utils/scc/scc.go` and naming the package `scc`, but the root dir will still be `utils`, wdyt?

Reply:
or `pkg/utils/openshift/scc.go` with the package name as `openshift`

Reply:
@concaf why `utils` ? it doesn't bring any value, any "function" is a "utils" (or helper) function for someone using it. `scc`, `securitycontext`, `openshift` are all decent name yes. It's then more about usage (`scc.GetClient`, vs `securitycontext.GetClient` vs `openshift.GetSecurityContextClient`) and package "size" (and dependency graph) than anything else 👼🏼

Reply:
fine, `common/openshift/scc.go` sounds better to me